addon/components/help-widget/keyboard-shortcuts/combo.hbs

```diff
@@ -0,0 +1,4 @@
+<span class="ember-help-widget-keyboard-combo">
+  {{#each @combo as |key|}}<kbd>{{key}}</kbd>{{/each}}
+</span>
+{{#if this.needsConnector}}&nbsp;{{t "ember-help-widget.or"}}&nbsp;{{/if}}
```

Comment:
⬇️  `&nbsp;` isn't the best when we need to adjust styling. I'd make a suggestion to use a `<span>` with a class or something.

Reply:
🤔 Yeah, I think it's because it's between two spans that the whitespace was being swallowed, but maybe I could just put it in a span and use regular whitespace? I tend to not like using `&nbsp;` either, but it also seems weird to have basic content spacing off in a CSS class somewhere. 😅

Reply:
I tried just doing `<span> {{t "ember-help-widget.or"}} </span>` and that seemed to do just fine in Chrome. 👍